File: python/Processor.py
class Processor:
    name = ""
    price = 0

    def __init__(self, name, price):
        self.name = name
        self.price = price

    def setName(self, name):
        self.name = name

    def getName(self):
        return self.name

    def setPrice(self, price):
        self.price = price

    def getPrice(self):
        return self.price

    def showProcessor(self):
        print("Processor name  : " + self.getName())
        print("Processor Price : " + str(self.getPrice()))

File: PHP/Processor.php
<?php
class Processor
{
    private $name;
    private $price;

    function __construct($name = "", $price = 0)
    {
        $this->name = $name;
        $this->price = $price;
    }

    function setName($name)
    {
        $this->name = $name;
    }

    function  getName()
    {
        return $this->name;
    }

    function setPrice($price)
    {
        $this->price = $price;
    }

    function getPrice()
    {
        return $this->price;
    }

    function showProcessor()
    {
        echo "Processor Name  : " . $this->getName() . "<br>";
        echo "Processor Price : " . $this->getPrice() . "<br>";
    }

    function __destruct(){}
}